content/operate/rs/references/rest-api/objects/cm_settings.md

```diff
@@ -0,0 +1,15 @@
+---
+Title: Cluster Manager settings object
+alwaysopen: false
+categories:
+- docs
+- operate
+- rs
+description: A REST API object that represents Cluster Manager UI settings
+linkTitle: cm_settings
+weight: $weight
```

Comment:
Shouldn't `weight` have a real value here?

Reply:
It just defaults to alphabetical in this case.